### ISSUE TITLE
Node dependency for upcoming Angular Material 1.1 release

### ISSUE THREAD

**Issue by cderamus** [daniel-nagy/md-data-table#434]
Are there any plans to update the project with support for the upcoming release of Angular Material (1.1)? There's obviously work arounds users can do, but for those who use webpack to manage and build dependencies it would be a nice addition. 

When attempting to build my app with the latest Angular Material release (1.1.0-RC5)  I receive the following:

npm ERR! peerinvalid Peer angular-material-data-table wants angular-material@^1.0.0

Thanks in advance!

**Comment by eladh** [comment]
Hi .
its really important - we want to use this amazing lib but when adding angular material 1.1.0-rc.5 the demo table dont display properly .


10x

**Comment by taxpayer** [comment]
@eladh, what does not display correctly when using Angular Material 1.1.0-rc5? I'm using it right now in one of my projects with md-data-table and it shows correctly.

**Comment by eladh** [comment]
![table](https://cloud.githubusercontent.com/assets/1949539/16451269/ac9d3afa-3e0a-11e6-8f67-d88a4fe5153d.jpg)

**Comment by eladh** [comment]
Im not getting any error msg and all files are loaded properly 
just switching back to 1.0.9 and the table working properly


10x

**Comment by taxpayer** [comment]
Strange @eladh, I'm using it and everything's working (Angular version 1.5.0)

![image](https://cloud.githubusercontent.com/assets/9285710/16455004/3cc1b11e-3de9-11e6-8ec6-e7a442264d50.png)

**Comment by nabblschnur** [comment]
Yes in package.json the peer dependency is tied to: ^1.0.0 angular material.

We need that changed to 1.1.0

**Comment by itspers** [comment]
Is there a way to install it with 1.1.0-RC5 in current state?

**Comment by Michsior14** [comment]
@itspers  Just install it normally, it will work despite of error.
